src/interactions/press/usePress.ts

```diff
@@ -185,6 +185,10 @@ export function usePress<T extends HTMLElement = HTMLElement>(
         detach()
       }
 
+      props.onMouseDown = (event) => {
```

Comment:
Can you comment this line for what we fixes?

Reply:
Done.